docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_CMS_Menu.md

```diff
@@ -107,6 +107,19 @@ class. For more information about configuring extensions see the
 LeftAndMain::add_extension('CustomLeftAndMain')
 ```
 
+## Customising the CMS help menu
+
+The CMS help menu links in the south toolbar are configurable via your [configuration file](../../configuration).
+You can edit existing or add new links as shown in the example below:
+
+```yml
+// app/_config/config.yml
+SilverStripe\Admin\LeftAndMain:
+  help_links:
+    'CMS User help': 'my-custom-userhelp-docs.com'
```

Comment:
Can we use proper URLs, ideally to `https://example.com` or `https://example.org` or `https://example.net`.

Some out there will copy paste this into their code and not realised they need the protocol for this to work.

Can we add an example illustrating how to suppress a link we don't want.